app/components/NestedCheckboxes.tsx

```diff
@@ -13,6 +13,7 @@ type CheckboxArgs = Parameters<typeof Checkbox>
 type CheckboxProps = CheckboxArgs[0]
 interface NestedCheckboxProps extends CheckboxProps {
   nodes: CheckboxProps[]
+  childoncheckhandler?: (arg: any) => void
```

Comment:
Specify the actual type of `arg` (a React ref).

Reply:
Updated